### DISCUSSION TITLE
运行ColossalAI的示例代码报错

### DISCUSSION THREAD

**Discussion by chinoll** [hpcaitech/ColossalAI#841] [Community | Q&A] [answered]
请问如何解决运行[AMP示例](https://github.com/hpcaitech/ColossalAI-Examples/blob/f447b18f01db230436c91c13ea772ea1d476d9c3/features/amp/train_with_engine.py)代码报错的问题？
报错内容
```
RuntimeError: CUDA error: the provided PTX was compiled with an unsupported toolchain.
CUDA kernel errors might be asynchronously reported at some other API call,so the stacktrace below might be incorrect.
For debugging consider passing CUDA_LAUNCH_BLOCKING=1
```
```
#运行的命令
python -m torch.distributed.launch --nproc_per_node 1 --master_addr localhost --master_port 29500 train_with_engine.py --config config/config_AMP_naive.py
```
pytorch version:1.10.2
colossalAI version:0.1.3
NVIDIA-SMI 460.106.00   Driver Version: 460.106.00   CUDA Version: 11.2

**Comment by FrankLeeeee** [comment, marked as answer]
Hi, this seems that there is a version mismatch according to [this port](https://forums.developer.nvidia.com/t/provided-ptx-was-compiled-with-an-unsupported-toolchain-error-using-cub/168292).
